### PR TITLE
Add Scheme Solution

### PR DESCRIPTION
**Pull request by William103**
## Description
Here is a solution in Scheme. I saw #78 is a pull request for a solution in Scheme, but it doesn't look like it's active. My solution is very similar, but differs in a few key ways. #78 used a boolean vector, which isn't necessarily a bit array, while I implemented a bit array. #78 was marked as not being faithful since it lacks a class to encapsulate the state of the sieve. While it is possible to create things like classes in Scheme, it isn't very idiomatic. Instead I created a library to encapsulate the state of the sieve. It accomplishes the same goal but is more idiomatic. If that isn't acceptable, I am happy to rewrite it to use a class.

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeScheme/solution_1/sieve.ss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  (define (validate-results n)
+    (= (cdr (assq n results-alist)) (count-primes n)))
+
+  ; convert scheme duration to fractional number of seconds
+  (define (duration-to-seconds duration)
+    (+ (time-second duration) (* 1e-9 (time-nanosecond duration))))
+
+  ; print the results in the expected format
+  (define (print-results duration passes n)
+    (format #t "William103;~a;~a;1;algorithm=base,bits=1" passes (duration-to-seconds duration))
```
Comment (rbergen) [inline PrimeScheme/solution_1/sieve.ss]: Is there a particular reason why you don't output a faithful tag? As indicated in the contributing guidelines, that means your solution will be considered to be unfaithful.

Comment (William103) [reply PrimeScheme/solution_1/sieve.ss]: I just wanted to make sure it actually was faithful before adding that tag. I changed it to output a faithful tag now.

File: PrimeScheme/solution_1/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    $ docker build -t scheme1 .
+    $ docker run --rm scheme1
+
+## Output
+
+Using chezscheme 9.5.4 on Arch Linux, I get
+
+    compiling sieve.ss with output to sieve.so
+    compiling PrimeScheme.ss with output to PrimeScheme.so
+    William103;141;5.0155461;1;algorithm=base,bits=1
```
Comment (rbergen) [inline PrimeScheme/solution_1/README.md]: It looks like you have not updated your example output to include the faithful tag. :) Anyway, I won't withhold approval of the PR on this.

Comment (William103) [reply PrimeScheme/solution_1/README.md]: Whoops! Updated that. Thanks!

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**